university-de-dschang.csl

```diff
@@ -0,0 +1,719 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded">
+  <info>
+    <title>University de Dschang (APA-modified)</title>
+    <title-short>UniDs</title-short>
+    <id>http://www.zotero.org/styles/university-de-dschang</id>
+    <link href="http://www.zotero.org/styles/university-de-dschang" rel="self"/>
+    <author>
```

Comment:
Add a link with rel="documentation" to the author guidelines.

Reply:
What do you mean? This is my first time attempting something like this.

Reply:
See points 8 and 10 here: https://github.com/citation-style-language/styles/wiki/Style-Requirements